CppCoreGuidelines.md

```diff
@@ -19153,6 +19154,10 @@ It is not standard and it is not portable.  It injects the hosting machine's fil
 into your program, in addition to locking you down to a vendor.
 Our recommendation is to write in ISO C++: See [rule P.2](#Rp-Cplusplus).
 
+You can also use `#pragma once` but fall back to the include guard when not compiling with the
```

Comment:
this would contradict the sentence directly above. See also #1019 for the longer discussion

Reply:
Removed it.